performance/performance_tuning.rst

```diff
@@ -49,12 +103,67 @@ Lastly, you can create indexes on your tables to enhance database performance. I
 
 For write performance, you can use general PostgreSQL configuration tuning to increase INSERT rates. We commonly recommend increasing checkpoint_timeout and max_wal_size settings. Also, depending on the reliability requirements of your application, you can choose to change fsync or synchronous_commit values.
 
+Once you have tuned a worker to your satisfaction you will have to manually apply those changes to the other workers as well. To verify that they are all behaving properly, set this configuration variable on the master:
+
+::
+
+  SET citus.explain_all_tasks = 1;
```

Comment:
@ozgune We already have a note in the last paragraph. @begriffs Could we make that into a proper "Note" form? So it stands out visually?